sled-agent/src/bootstrap/params.rs

```diff
@@ -14,7 +14,7 @@ use serde::{Deserialize, Serialize};
 use sha3::{Digest, Sha3_256};
 use sled_hardware::Baseboard;
 use std::borrow::Cow;
-use std::collections::HashSet;
+use std::collections::BTreeSet;
```

Comment:
Ah, is this for consistent order?

Reply:
Yeah, for the expectorate test I added with madrid's config. With `HashSet`/`HashMap` it's a lot more annoying to check the parsed value.